src/site/content/en/blog/vitals-tools-workflow/index.md

```diff
@@ -0,0 +1,507 @@
+
+The overall Lighthouse score for the URL displays at the top of the page.
+Below the field data section is the lab data section
+that provides metrics collected from a real-time Lighthouse run
+and surfaces opportunities for the page.
+
+Running PSI for different types of pages—analyzing the
+field data and opportunities—will provide you with
+a high-level understanding of metrics and improvements to focus on.
+With that information,
+you are now ready to debug and optimize your pages which we will discuss in the next section.
+
+## Step B: Debug and optimize
+
+{% Img src="image/ZDZVuXt6QqfXtxkpXcPGfnygYjd2/qWh67N9ZOLOu12FzO7oz.png",
+alt="Image showing the tools used in this step: Lighthouse, Web Vitals Extension, Chrome DevTools.",
+width="800", height="188" %}
+
+1. Use [Lighthouse](/lighthouse-performance/)
+to audit a page and uncover improvements.
+2. Use the
+[Web Vitals extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
+to analyze CWV metrics in real-time on a page.
+3. Use [Chrome DevTools](https://developer.chrome.com/docs/devtools/)
+to debug performance issues and test code changes.
+
+Once you have identified the pages and metrics that need attention,
+the next questions to ask are:
+
+- How can these pages be optimized?
+- What are the low-hanging fruits to tackle immediately?
+- What improvements require more planning?
+
+### Uncover opportunities with Lighthouse
+
+The first step to answer these questions is using Lighthouse.
+It provides a snapshot of opportunities to improve the load performance
+for specific pages based on a lab run.
+You can access Lighthouse in several ways, including:
+
+- Running
+[Lighthouse from Chrome DevTools](https://developers.google.com/web/tools/lighthouse#devtools).
+- At [web.dev/measure](/measure/).
+- From [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/).
+
+Lighthouse identifies critical areas of optimization at load time
+and by how much they could potentially improve the user experience.
+Quickly identifying low-effort fixes helps to start the optimization process
+before handling the bigger tasks.
+
+While the metrics' absolute values may be interesting to look at,
+do keep in mind that they change based on the environment Lighthouse is running in
+and many other factors,
+such as the network conditions of the servers hosting the website and its third parties.
+In addition,
+Lighthouse simulates a user's experience on a mid-tier mobile on a slow-4G connection,
+and might not represent your users' experience at the 75th percentile
+(which the CWV metrics are based on).
+The metrics are a good indication of what is going on in a website,
+but we recommend focusing more on the optimization opportunities in the report.
+
+{% Aside %}
+Lighthouse analyzes pages at load time in a lab environment.
+While lab assessments of LCP are often useful to debug and optimize LCP in the field,
```

Comment:
```suggestion
While lab assessments of LCP are often useful to debugging and optimizing LCP in the field,
```

Reply:
would that not be "useful for debugging..."? "Useful to debugging" sounds odd.